label_sync/labels.md

```diff
@@ -0,0 +1,124 @@
+
+### How do I add a new label?
+
+- Add automation that consumes/produces the label
+- Open a PR against [labels.yaml](https://git.k8s.io/test-infra/label_sync/labels.yaml)
+- Involve [sig-contributor-experience](https://git.k8s.io/community/sig-contributor-experience) in the change, eg: chat about it in slack, mention @kubernetes/sig-contributor-experience-pr-reviews in the PR, etc.
+- After the PR is merged, a kubernetes CronJob is responsible for syncing labels daily
+
+
+## Labels that apply to both
+
+| Name | Description | Added By | Prow Plugin |
+| ---- | ----------- | -------- | --- |
+| <a id="close/duplicate" href="#close/duplicate">`close/duplicate`</a> | TODO, see [kubernetes/test-infra#7251](https://github.com/kubernetes/test-infra/issues/7251)| humans | |
+| <a id="close/needs-information" href="#close/needs-information">`close/needs-information`</a> | TODO, see [kubernetes/test-infra#7251](https://github.com/kubernetes/test-infra/issues/7251)| humans | |
+| <a id="close/not-reproducible" href="#close/not-reproducible">`close/not-reproducible`</a> | TODO, see [kubernetes/test-infra#7251](https://github.com/kubernetes/test-infra/issues/7251)| humans | |
+| <a id="close/unresolved" href="#close/unresolved">`close/unresolved`</a> | TODO, see [kubernetes/test-infra#7251](https://github.com/kubernetes/test-infra/issues/7251)| humans | |
+| <a id="committee/conduct" href="#committee/conduct">`committee/conduct`</a> | Denotes an issue or PR intended to be handled by the (as of yet non-existent) code of conduct committee.| anyone |  [label](https://git.k8s.io/test-infra/prow/plugins/label) |
+| <a id="committee/steering" href="#committee/steering">`committee/steering`</a> | Denotes an issue or PR intended to be handled by the kubernetes steering committee.| anyone |  [label](https://git.k8s.io/test-infra/prow/plugins/label) |
+| <a id="kind/bug" href="#kind/bug">`kind/bug`</a> | Categorizes issue or PR as related to a bug.| anyone |  [label](https://git.k8s.io/test-infra/prow/plugins/label) |
+| <a id="kind/documentation" href="#kind/documentation">`kind/documentation`</a> | Categorizes issue or PR as related to documentation.| anyone |  [label](https://git.k8s.io/test-infra/prow/plugins/label) |
+| <a id="kind/feature" href="#kind/feature">`kind/feature`</a> | Categorizes issue or PR as related to a new feature.| anyone |  [label](https://git.k8s.io/test-infra/prow/plugins/label) |
+| <a id="lifecycle/frozen" href="#lifecycle/frozen">`lifecycle/frozen`</a> | Indicates that an issue or PR should not be auto-closed due to staleness. <br><br> This was previously `keep-open`, | anyone |  [lifecycle](https://git.k8s.io/test-infra/prow/plugins/lifecycle) |
+| <a id="lifecycle/rotten" href="#lifecycle/rotten">`lifecycle/rotten`</a> | Denotes an issue or PR that has aged beyond lifecycle/stale and will be auto-closed.| fejta-bot or anyone |  [lifecycle](https://git.k8s.io/test-infra/prow/plugins/lifecycle) |
```

Comment:
why does this list fejta bot, when stale doesn't?

Reply:
good catch